### ISSUE TITLE
vis_cam.py in mmsegmenation

### ISSUE THREAD

**Issue by yuanyuangoo** [open-mmlab/mmsegmentation#2032]
a grad-cam-like tool to visualize feature maps, like in mmclassification

**Comment by MeowZheng** [comment]
of course, we have supported very powerful visualization tool in mmseg 1.0.0rc0, you can try it

reference 
https://github.com/open-mmlab/mmengine/blob/cfb884c180e97454b14b1dc12bac21bd376f82db/mmengine/visualization/visualizer.py#L119-L124
https://github.com/open-mmlab/mmsegmentation/blob/1.x/docs/en/user_guides/visualization.md

**Comment by yuanyuangoo** [comment]
Thanks for the response. How do I get a feat-map of a certain layer after I trained a model using mmsegmentation and got weights?

**Comment by MeowZheng** [comment]
Just save any tensort map like below as you want, and load it then see it.

https://github.com/open-mmlab/mmsegmentation/blob/b51670b61339e5b10c5ab6c277de6b6a387fdff0/mmseg/models/decode_heads/dpt_head.py#L287

**Comment by shigejiafeimao** [comment]
> Just save any tensor map like below as you want, then load it ant see it with the visualization tool.
> 
> https://github.com/open-mmlab/mmsegmentation/blob/b51670b61339e5b10c5ab6c277de6b6a387fdff0/mmseg/models/decode_heads/dpt_head.py#L287

could you speak in details? I also want to know. please!

**Comment by shigejiafeimao** [comment]
> Just save any tensor map like below as you want, then load it ant see it with the visualization tool.
> 
> https://github.com/open-mmlab/mmsegmentation/blob/b51670b61339e5b10c5ab6c277de6b6a387fdff0/mmseg/models/decode_heads/dpt_head.py#L287

if i use sep_aspp_head.py，how to get a feat-map?

**Comment by duduzai2019** [comment]
> Can we have a grad-cam-like tool to visualize feature maps, like in vis_cam.py in misclassification?

Hi, have you solved it?

**Comment by duduzai2019** [comment]
> Hi, have you solved it?

Hi, can you solved it?

**Comment by shenxiangkei** [comment]
Have you found a suitable solution, I have a similar problem
